cmd/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"cmp"
+	"log"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/tmavrin/mock-http-server/internal/config"
+	"github.com/tmavrin/mock-http-server/internal/mock"
+)
+
+func main() {
+	cfg := configPrepare()
+
+	server := http.Server{
+		Addr:              ":8080",
```

Comment:
```suggestion
		Addr:              fmt.Sprintf(":%s", cmp.Or(os.Getenv("PORT"), "8080")),
```

To enable `PORT` configuration with env

Reply:
done